src/main/java/com/bloomtech/asylumrgbea/models/AsylumCaseRequestDto.java

```diff
@@ -1,8 +1,17 @@
 package com.bloomtech.asylumrgbea.models;
 
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
-@Data
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
 public class AsylumCaseRequestDto {
+
+    private Integer numberOfItemsInPage;
```

Comment:
It is a common practice to check for null anything that needs a default value has a default value. Null triggers an exception that passes a message to the front end. One reason it is best practice is because a default value can be modified and null can not. That would present logic errors and the program would continue to execute creating many branches of problems in the data that would later have to be fixed. Compare that to a handled and logged null pointer that stops execution immediately.